plugins/secretstores/docker/README.md

```diff
@@ -0,0 +1,26 @@
+# Docker Secrets Secret-Store Plugin
```

Comment:
Should we maybe put a link to https://docs.docker.com/engine/swarm/secrets/ in the readme to give users a good starting point?

Reply:
Yes, I will add some relevant links to Docker Swarm Secrets as well as Documentation for Standalone Docker Compose deployment.